### DISCUSSION TITLE
Power Apps Development Kick-off!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/Power-Platform-For-Beginners#4] [Power Apps Development]
Do you have a section of the [template](https://github.com/microsoft/Power-Platform-For-Beginners/tree/main/templates) you would like to start working on?
Let your teammates know!

Then you can **Fork** the repo and start creating content!

**Comment by LuiseFreese** [comment]
Hey there ;-)

**Comment by angelogulisano** [comment]
Hello, I'm here :-)

**Comment by RebekkaAalbers** [comment]
Hi all!
He @LuiseFreese and @angelogulisano 👋.
@FBoucher, is this going to be the team for the Power Apps development part? If so I suggest to setup a call with the 3 of us to structure things a bit and share here the outcome so you are informed. What do you think @LuiseFreese and @angelogulisano?

**Reply by angelogulisano** [reply]
Hello @RebekkaAalbers , yes! I'm available tomorrow or next week. Let me know :-)

**Reply by RebekkaAalbers** [reply]
Let's add @AhmedSalih-PowerPlatformPlace to the conversation. @AhmedSalih-PowerPlatformPlace, my suggestion is to have a call with all contributors to make an outline of what should be in this section and than assign people to different topics. 
Are you available next week? And if so what time?

My availability for next week is in CEST timezone:
- Monday August 21 Afternoon of evening
- Tuesday August 22 Afternoon
- Wednesday August 23 Afternoon
- Friday August 25 Afternoon

**Reply by angelogulisano** [reply]
My availability for next week are:

- Monday August 21 morning, Afternoon of evening
- Tuesday August 22, morning from 11.00am to 1pm Afternoon from 4.00 pm
- Wednesday August 23 morning, Afternoon of evening
- Friday August 25 morning, Afternoon

thanks

**Reply by LuiseFreese** [reply]
hey there, 

I can make time

* Monday, Aug 21, 11-14 or 16-18
* Tuesday, Aug 22, 13 - 18
* Wednesday, Aug 23, 12 -17
* Friday, Aug 25, 14-18

**Reply by AhmedSalih-PowerPlatformPlace** [reply]
My availability for next week is:

- Monday August 21 Evening PT
- Tuesday August 22, Evening PT
- Wednesday August 23 morning, Afternoon to 3PM PT
- Thursday August 24 morning, Afternoon, and Evening CST
- Friday August 25 morning, Afternoon, and Evening CST

**Reply by RebekkaAalbers** [reply]
I'm scheduling something for Friday August 25th 16:00-17:00 CEST / 9-10AM CST. 
Can you all DM me the email address where I can send the invite?
Thanks!

**Reply by RebekkaAalbers** [reply]
You should all have received an invite for Friday.

**Comment by AhmedSalih-PowerPlatformPlace** [comment]
Hey Frank, I would love to work on this topic covering everything we need about Dataverse and MDA. I will also work on the Power Pages if that is okay, but my expertise, is Dataverse, MDA, Power Fx, and JavaScript, all together for the full power of the Model-Driven Apps 😁

**Reply by fboucher** [reply]
@AhmedSalih-PowerPlatformPlace sure

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)
